src/test/game-mode.test.ts

```diff
@@ -41,4 +43,42 @@ describe("game-mode", () => {
       expect(classicGameMode.isWaveTrainer(19, arena)).toBeFalsy();
     });
   });
+  /*
+  // Need to figure out how to properly start a battle
+  describe("Luck Check", async () => {
+    let classicGameMode: GameMode;
+    let dailyGameMode: GameMode;
+
+    beforeAll(() => {
+      phaserGame = new Phaser.Game({
+        type: Phaser.HEADLESS,
+      });
+    });
+
+    afterEach(() => {
+      game.phaseInterceptor.restoreOg();
+    });
+
+    beforeEach(() => {
+      game = new GameManager(phaserGame);
+      classicGameMode = getGameMode(GameModes.CLASSIC);
+      dailyGameMode = getGameMode(GameModes.DAILY);
+    });
+
+    it("applies luck in Classic", () => {
+      game.override
+        .shinyLevel(true, 2);
+      game.classicMode.startBattle([Species.PICHU]);
+      const party = game.scene.getParty();
+      expect(getPartyLuckValue(party)).toBe(3);
+    });
+    it("does not apply luck in Daily Runs", () => {
+      game.override
+        .shinyLevel(true, 2);
+      game.dailyMode.startBattle();
+      const party = game.scene.getParty();
+      expect(getPartyLuckValue(party)).toBe(0);
+    });
+  });
+  //*/
```

Comment:
Jut add a `.skip` to the describe if you don't want to run it (yet)

```suggestion
  // Need to figure out how to properly start a battle
  describe.skip("Luck Check", async () => {
    let classicGameMode: GameMode;
    let dailyGameMode: GameMode;

    beforeAll(() => {
      phaserGame = new Phaser.Game({
        type: Phaser.HEADLESS,
      });
    });

    afterEach(() => {
      game.phaseInterceptor.restoreOg();
    });

    beforeEach(() => {
      game = new GameManager(phaserGame);
      classicGameMode = getGameMode(GameModes.CLASSIC);
      dailyGameMode = getGameMode(GameModes.DAILY);
    });

    it("applies luck in Classic", () => {
      game.override
        .shinyLevel(true, 2);
      game.classicMode.startBattle([Species.PICHU]);
      const party = game.scene.getParty();
      expect(getPartyLuckValue(party)).toBe(3);
    });
    it("does not apply luck in Daily Runs", () => {
      game.override
        .shinyLevel(true, 2);
      game.dailyMode.startBattle();
      const party = game.scene.getParty();
      expect(getPartyLuckValue(party)).toBe(0);
    });
  });
```

Reply:
Okay, thanks